neard/src/cli.rs

```diff
@@ -58,7 +58,7 @@ impl NeardCmd {
             }
         }
```

Comment:
Should we move these, or at least the version log, to after the global subscriber has been initialized?